apps/accounts/forms.py

```diff
@@ -86,3 +87,11 @@ def clean_confirm_password(self):
                 )
         password_validation.validate_password(confirm_password)
         return confirm_password
+
+
+class LoginForm(BaseAuthenticatonForm):
+
+    def __init__(self, request=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
As you've gone crispy on this form, we might need something for the `next` url, eg.

https://github.com/developersociety/achurchnearyou/blob/master/templates/accounts/login.html

Otherwise login protected urls where they set `?next=/path/to/view/` will just redirect to the home page instead.

I think you can add bits of django template into the form here.

Reply:
Don't think I want this I was planning to use https://docs.djangoproject.com/en/2.0/ref/settings/#login-redirect-url

Reply:
Problem is that `AuthenticationForm` doesn't have a `next` field, but `LoginView` can use the `next` GET parameter to redirect you to the right URL, if that makes sense?

I think you need:

```
HTML("""
    {% if next %}<input type="hidden" name="next" value="{{ next }}">{% endif %}
""")
``` 

Here?

Reply:
Right, sorry I was thinking about redirect after successful login not the one with requested URL.

Reply:
Updated